src/services/ExpressionBuilder.ts

```diff
@@ -0,0 +1,636 @@
+import { AdHocFilterWithLabels, sceneUtils } from '@grafana/scenes';
```

Comment:
Can you please add unit tests for this new service?

Reply:
Tests were in query.test.ts, moved to `ExpressionBuilder.test.ts` in https://github.com/grafana/explore-logs/pull/1045/commits/98fd579bc89becd59dee4907a8727be2c01d32eb

Reply:
Ah, perfect, thanks!